mlflow/tensorflow/__init__.py

```diff
@@ -887,14 +897,109 @@ def _patch_implementation(
                     # & tf.keras callbacks if necessary
                     callbacks = list(kwargs.get("callbacks") or [])
                     kwargs["callbacks"], self.log_dir = _setup_callbacks(
-                        callbacks, log_models, metrics_logger
+                        callbacks, False, metrics_logger
```

Comment:
@bali0019 @BenWilson2 Does this imply that there's some cleanup we can do within `_setup_callbacks` to remove model logging code, since we're always passing `False` now? Can we file a follow-up PR for a cleanup here?

Reply:
for TF <2.1.0 the fit_generator() method patch still uses this flag. Should we mark this with a remove note when we no longer support those older TF versions?

Reply:
Does that imply that our autologging for signatures / input examples doesn't work with `fit_generator()`. If so, why not?